micrometer-tracing-bridges/micrometer-tracing-bridge-otel/src/main/java/io/micrometer/tracing/otel/bridge/Slf4JEventListener.java

```diff
@@ -29,28 +29,45 @@ public class Slf4JEventListener implements EventListener {
 
     private static final InternalLogger log = InternalLoggerFactory.getInstance(Slf4JEventListener.class);
 
+    private static final String DEFAULT_TRACE_ID_KEY = "traceId";
+
+    private static final String DEFAULT_SPAN_ID_KEY = "spanId";
+
+    private final String traceIdKey;
+
+    private final String spanIdKey;
+
+    public Slf4JEventListener() {
+        this(DEFAULT_TRACE_ID_KEY, DEFAULT_SPAN_ID_KEY);
+    }
+
+    public Slf4JEventListener(String traceIdKey, String spanIdKey) {
```

Comment:
Could you please add Javadoc to this that includes a `@since 1.11.0` tag?

Reply:
BTW I don't think this is configurable for Brave so I doubt that this would be actually supported in Boot cause we want to have pretty much a parity between OTel and Brave set ups

Reply:
> BTW I don't think this is configurable for Brave so I doubt that this would be actually supported in Boot cause we want to have pretty much a parity between OTel and Brave set ups

Agree!